packages/gatsby-recipes/src/renderer/render.js

```diff
@@ -151,14 +153,17 @@ const handleResource = (resourceName, context, props) => {
   }
 
   let allResources = useResourceContext()
-  const error = validateResource(resourceName, context, props)
-  if (error) {
-    const result = {
-      error: `Validation error: ${error.details[0].message}`,
+  if (!errorCache.has(trueKey)) {
```

Comment:
cache validation

Reply:
The `trueKey` vs `cacheKey` is a bit confusing and I can't really tell why you're using one over the other here.

Reply:
`cacheKey` includes what the operation on the resource is — e.g. plan vs. create

`trueKey` is because we're currently juggling two ways of creating a resource ID — a user-set key or an automatically generated UUID — we need to unify internally but in the meantime we're using both